sync_radar_data_globus.py

```diff
@@ -120,14 +113,16 @@ def __init__(self, client_id, client_secret=None, transfer_rt=None):
         parser.add_argument("-t", "--data_type",
                             help="One of {} Default: 'raw'".format(self.possible_data_types),
                             default='raw')
+
```

Comment:
line 97: this might be nice as a constant list at the top for quick modifications, however, maybe a comment on what the acrnyms mean. Also does fit inlcude lmfit? and what about convection plots?